packages/abort-controller/tsconfig.cjs.json

```
@@ -1,16 +1,18 @@
 {
   "compilerOptions": {
-    "module": "commonjs",
-    "target": "es5",
     "lib": ["es5", "es2015.collection"],
     "strict": true,
-    "sourceMap": true,
+    "sourceMap": false,
     "declaration": true,
     "stripInternal": true,
     "rootDir": "./src",
     "outDir": "./build",
```

Standardize outDir across `clients/*` and `packages/*`

Recommending `"dist/cjs"` and `"dist/es"` as:
* It's currently used in clients https://github.com/aws/aws-sdk-js-v3/blob/bddeb3d172f1e26ff9e429140333d6ea624168c2/clients/client-accessanalyzer/tsconfig.json#L15
* It's easy to have transpiled code in `dist` folder as a standard. In future, if we decide to create new bundle for `es5`, it can be put in `dist/es5` folder.

```suggestion
    "outDir": "dist/cjs",
```

---

Yeah this is a very good point. I dont really like build and build-es very much

---

done, everything should be standardized to cjs. All packages should now emit a es and cjs bundle, module will map to es